.build/external-documentation/cli-getting-started.quickstart.yml

```diff
@@ -4,6 +4,14 @@ metadata:
   name: docs-cli-getting-started
   annotations:
     order: 1
+  labels:
+    service-category/application-services: true
+    product/rhosak: true
+    product/rhosr: true    
+    product/rhoad: true
+    product/rhoam: true
```

Comment:
Don't think this guide applies to RHOAM?

Reply:
+1 Yes, we can remove RHOAM.

Reply:
removed